### PR TITLE
Add a page on the upcoming key signing party

### PR DESCRIPTION
**Pull request by daenney**
Paging @nibalizer for a verification of the content and suggested procedure.

### CODE REVIEW

**Comment by KrisBuytaert** [comment]
Waiting for @nibalizer to verify content :)

File: ksp.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        the Web of Trust (WoT) and allow community members that are
+        likely already in contact with each other to exchange signatures.</p>
+
+        <p>If you've ever been to a KSP before (like the one at FOSDEM) you'll
+        be familiar with the protocol. Here's how this is going to work:</p>
+        <ul>
+          <li>Upload your key to the <a href="https://sks-keyservers.net">GnuPG/sks-keyservers</a> pool of keyservers</li>
+          <li>Send an email to cfgmgmtcampksp@tutanota.com with:
+            <ul>
+              <li>subject: KSP: &lt;name&gt; &lt;0xlongkeyid&gt; &lt;fingerprint&gt;</li>
```
Comment (nibalizer) [inline ksp.html]: Overloading the subject like this doesn't make sense
Make the subject be 'KSP'
and the email contain the output of `gpg --fingerprint mykey`

File: ksp.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+              <li>deadline: Wednesday the 79th of January 2016</li>
```
Comment (nibalizer) [inline ksp.html]: The 79th of january.. interesting

File: ksp.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            </ul></li>
+          <li>You bring with you on the day of the KSP:
+            <ul>
+              <li>One government issued photo ID (passport or ID, drivers license does not count).</li>
+              <li><strong>Optional:</strong> One other form of ID (drivers license, security badge, etc.).</li>
+              <li>A piece of paper with your &lt;0xlongkeyid&gt;, &lt;fingerprint&gt; and &lt;keysize&gt;.</li>
+              <li>Two pens, one for you and one for the person next to you that will have forgotten to bring theirs.</li>
+              <li>A copy of the document we'll use at the KSP and the <code>SHA256</code> hash of that document that you will compute yourself. We will provide the final list on Friday the 29th of January. Watch inbox and Twitter for the URL.</li>
```
Comment (nibalizer) [inline ksp.html]: We'll also post a link here

File: ksp.html (hunk 1 above)
Comment (daenney) [inline ksp.html]: I've fixed this but Github still wants to show the diff, for whatever reason.

**Comment by nibalizer** [comment]
Looks good. Shipit

**Pull request merged**